ArcGISRuntimeSDKQt_QMLSamples/Layers/ExportVectorTiles/ExportVectorTiles.qml

```diff
@@ -0,0 +1,275 @@
+// [WriteFile Name=ExportVectorTiles, Category=Layers]
+// [Legal]
+// Copyright 2022 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.12
+import Esri.ArcGISRuntime 100.15
+import QtQuick.Controls 2.12
+import QtQuick.Layouts 1.15
+import Esri.ArcGISExtras 1.1
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    property int exportProgress: 0
+    property int exportJobStatus: 0
+
+    // add a mapView component
+    MapView {
+        id: mapView
+        anchors.fill: parent
+
+        Map {
+            id: map
+            initBasemapStyle: Enums.BasemapStyleArcGISStreetsNight
+            initialViewpoint: ViewpointCenter {
+                id: viewpoint
+                Point {
+                    x: -117.181
+                    y: 34.049
+                    spatialReference: SpatialReference { wkid: 4326 }
+                }
+                targetScale: 1e4
+            }
+        }
+
+
+
+        GraphicsOverlay {
+            id: graphicsOverlay
+
+            Graphic {
+                id: exportAreaGraphic
+                symbol: SimpleLineSymbol {
+                    style: Enums.SimpleLineSymbolStyleSolid
+                    color: "green"
+                    width: 3
+                }
+            }
+        }
+
+        Component.onCompleted: {
+            // Set and keep the focus on MapView to enable keyboard navigation
+            forceActiveFocus();
+        }
+    }
+
+    Rectangle {
+        id: exportProgressWindow
+        anchors.centerIn: parent
+
+        color: "white"
+        visible: exportJobStatus !== 0 && exportJobStatus !== 3 && exportJobStatus !== 4
```

Comment:
Either add a comment or use variables to describe which state each number indicates.